xwiki-platform-tools/xwiki-platform-tool-configuration-resources/src/main/resources/hibernate.cfg.xml.vm

```diff
@@ -304,6 +304,10 @@ jdbc:hsqldb:file:${environment.permanentDirectory}/database/xwiki_db;shutdown=tr
     <property name="hibernate.connection.useUnicode">true</property>
     <property name="hibernate.connection.characterEncoding">utf8</property>
 
+    <!-- To enable if basic compression feature is enabled in Oracle
+    <property name="xwiki.compressionAllowed">true</property>
+    -->
```

Comment:
Hmmm this comment is not good, it's already inside another comment so it's breaking the file. Going to fix this.

Reply:
I actually chose to always put the property but with a false value, and add the info in the notes, see: https://github.com/xwiki/xwiki-platform/commit/72e10a13c69e7b051c9e14f1fe71ff195a86b58b. Should work, no?

Reply:
I commented your change, IMO this is a bad idea.